### PR TITLE
Dev blog post for Opera 52 - paintlets.

### PR DESCRIPTION
**Pull request by bratell-at-opera**
There is a couple of weeks until the likely release date so no urgency.

### CODE REVIEW

File: src/blog/_posts/2018-04-01-opera-52.md [hunk 1]
```diff
@@ -0,0 +1,155 @@
...
+  on `<slot>` elements, which acts
+  like `assignedNodes()` but only returns element nodes.
+* [`HTMLAnchorElement.relList`](https://html.spec.whatwg.org/#dom-link-rellist)
+  will now give an accurate mapping of the `rel` attribute.
+* [`document.all`](https://html.spec.whatwg.org/#htmlallcollection)
+  can no longer be overwritten.
+
+## Feature Policy
+* A new [feature policy](https://wicg.github.io/feature-policy/)
+  `sync-xhr` willallow a site to block synchronous
```
Comment (jensl) [inline src/blog/_posts/2018-04-01-opera-52.md]: Missing space in "willallow".

File: src/blog/_posts/2018-04-01-opera-52.md [hunk 2]
```diff
@@ -0,0 +1,155 @@
...
+  server has responded which can make them harmful for the
+  experience. If a site embeds untrusted, possibly malicious, content
+  (think ads), this adds a tool to lock that content down a bit.
+
+## Network
+* The https code is updated to match the [draft-23
+  version](https://tools.ietf.org/html/draft-ietf-tls-tls13-23) of the
+  TLS protocol.
+* [`Request.destination`](https://fetch.spec.whatwg.org/#concept-request-destination)
+  is added to give service workers a better understand on why a resource
```
Comment (jensl) [inline src/blog/_posts/2018-04-01-opera-52.md]: "understand on" => "understanding of"? And maybe "is being fetched".

File: src/blog/_posts/2018-04-01-opera-52.md [hunk 3]
```diff
@@ -0,0 +1,155 @@
...
+
+## Performance APIs
+* A `toJSON()` method has been added to `PerformanceResourceTiming`,
+  `PerformanceLongTaskTiming` and `TaskAttributionTiming`.
+
+## Security
+* The [download](https://html.spec.whatwg.org/#attr-hyperlink-download)
+  attribute will be ignored on anchor elements with
+  cross-origin attributes.
+* Certificates from Symantec's Legacy PKI issued after 1 December,
```
Comment (jensl) [inline src/blog/_posts/2018-04-01-opera-52.md]: I think dates are commonly formatted "1 December 2017" or "December 1, 2017". Former commonly on this side of the Atlantic ocean and latter on the other side, apparently. If you had used the latter here, I probably wouldn't have thought about it. With this spelling, I ended up reading Wikipedia articles about it. I'm pretty sure the world will end if we get it wrong.

File: src/blog/_posts/2018-04-01-opera-52.md [hunk 4]
```diff
@@ -0,0 +1,155 @@
...
+    <script>
+      CSS.paintWorklet.addModule('circle_arc.js');
+    </script>
+
+The module itself is defined with the javascript below.
+
+    // circle_arc.js
+    class CircleArcPainter {
+        circle_arc(ctx, x, y, radius, angle) {
+            let angleInRad =  (angle * 0.9 / 2 + 10) * Math.PI / 180;
```
Comment (fsoder) [inline src/blog/_posts/2018-04-01-opera-52.md]: Excess space after =.

File: src/blog/_posts/2018-04-01-opera-52.md [hunk 5]
```diff
@@ -0,0 +1,155 @@
...
+    }
+
+    // Register our class under a specific name
+    registerPaint('circlearcpainter', CircleArcPainter);
+
+
+## Server Timing API
+
+With the new [Server Timing API](https://w3c.github.io/server-timing/)
+there is a well defined way for servers pass performance information
```
Comment (fsoder) [inline src/blog/_posts/2018-04-01-opera-52.md]: ...for servers **to** pass performance...

File: src/blog/_posts/2018-04-01-opera-52.md [hunk 6]
```diff
@@ -0,0 +1,155 @@
...
+can make even better informed decisions.
+
+In Developer Tools this information is displayed in the Network view
+by selecting the resource and activating the Timing tab.
+
+## CSS
+* The
+  [`:any-link`](https://developer.mozilla.org/en-US/docs/Web/CSS/:any-link)
+  pseudo selector can be used to style both visited and non-visited
+  links at the same time.  For colors, the `rgb` and `hsl` functions
```
Comment (fsoder) [inline src/blog/_posts/2018-04-01-opera-52.md]: The change to `rgb()` et al ought to be a bullet on its own?

File: src/blog/_posts/2018-04-01-opera-52.md [hunk 7]
```diff
@@ -0,0 +1,155 @@
...
+  will now give an accurate mapping of the `rel` attribute.
+* [`document.all`](https://html.spec.whatwg.org/#htmlallcollection)
+  can no longer be overwritten.
+
+## Feature Policy
+* A new [feature policy](https://wicg.github.io/feature-policy/)
+  `sync-xhr` willallow a site to block synchronous
+  XMLHttpRequests. Synchronous XMLHttpRequests block scripts until a
+  server has responded which can make them harmful for the
+  experience. If a site embeds untrusted, possibly malicious, content
```
Comment (fsoder) [inline src/blog/_posts/2018-04-01-opera-52.md]: ...**user** experience...?

File: src/blog/_posts/2018-04-01-opera-52.md [hunk 8]
```diff
@@ -0,0 +1,155 @@
...
+  (think ads), this adds a tool to lock that content down a bit.
+
+## Network
+* The https code is updated to match the [draft-23
```
Comment (fsoder) [inline src/blog/_posts/2018-04-01-opera-52.md]: HTTPS? (Like TLS later in the sentence.)

**Pull request merged**